src/content/development/backports/_index.en.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Backports"
+date: 2021-06-21T16:03:26+02:00
+weight: 5
+---
+
+Fixes for serious issues or regressions affecting previous releases may be backported to the corresponding branches,
+to be included in the next release from that branch.
+
+## Requesting a backport
+
+Backports can only be requested on fixes made against a later branch, or the `devel` branch.
+(This doesn’t mean that bugs can’t be fixed in older branches directly; but where relevant, they should first be fixed on `devel`.)
+
+To request such a backport, identify the relevant pull request, and add the “backport” label to it.
+You should also add a comment to the pull request explaining why the backport is necessary, and which branch(es) are targeted.
+Issues should _not_ be labeled, they are liable to be overlooked or lack a one-to-one mapping to a code fix.
+
+## Handling backports
+
+Pending backports can be identified using
+[this query, listing all non-archived pull requests with a “backport” label and without a “backport-handled” label](https://github.com/pulls?q=is%3Apr+archived%3Afalse+user%3Asubmariner-io+label%3Abackport+-label%3Abackport-handled).
```

Comment:
This is a that long link:

```suggestion
[this query](https://github.com/pulls?q=is%3Apr+archived%3Afalse+user%3Asubmariner-io+label%3Abackport+-label%3Abackport-handled) that lists all non-archived pull requests with a “backport” label and without a “backport-handled” label.
```

Reply:
“this query” isn’t a descriptive link; descriptive links are important for accessibility (_e.g._ when using a screen reader).